code/drasil-code/Data/Drasil/ExternalLibraries/ODELibraries.hs

```diff
@@ -605,7 +605,7 @@ odeMethodUnavailable = "Chosen ODE solving method is not available" ++
 
 -- | Change in @X@ chunk constructor (where @X@ is a given argument).
 diffCodeChunk :: CodeVarChunk -> CodeVarChunk
-diffCodeChunk c = quantvar $ implVar' ("d" ++ c ^. uid)
+diffCodeChunk c = quantvar $ implVar' ("d" ++ showUID c )
```

Comment:
Hmm, I don't like this inconsistency. I think we should uniformly append. Hopefully fixing that now won't break things...